### PR TITLE
Fixed ProfileView and transitions

### PR DESCRIPTION
**Pull request by Alvareon**
- added ProfileView as a panel;
- transitions were fixed ;
- fixed "Back to activity" button;
backbtn

### CODE REVIEW

File: Assets/MirageXR/Tests/NewUI/RootView_v2.cs [hunk 1]
```diff
@@ -125,19 +122,19 @@ private void OnStepsClick(bool value)
         if (value) _pageView.currentPageIndex = 0;
     }
 
-    private void OnViewClick(bool value)
+    private void OnViewClick()
     {
-        if (value) _pageView.currentPageIndex = 1;
+        PopupsViewer.Instance.Show(_profilePrefab);
     }
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/RootView_v2.cs]: change `onViewClick` to `OnProfileClick` (or `OnProfileClicked`)

File: Assets/MirageXR/Tests/NewUI/Prefabs/Profile.prefab [hunk 2]
```diff
@@ -523,13 +523,13 @@ MonoBehaviour:
   m_OnClick:
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/Prefabs/Profile.prefab:523]: image

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ProfileView : PopupBase
+{
+    [SerializeField] private MoodleServersView _moodlePrefab;
+    [SerializeField] private Button _selectServer;
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/ProfileView.cs]: let's add the prefix `_btn` to the button links

File: Assets/MirageXR/Tests/NewUI/RootView_v2.cs [hunk 4]
```diff
@@ -9,12 +9,13 @@ public class RootView_v2 : BaseView
     public static RootView_v2 Instance { get; private set; }
 
     [SerializeField] private Toggle _toggleHome;
-    [SerializeField] private Toggle _toggleProfile;
+    [SerializeField] private Button _buttonProfile;
```
Comment (cawke) [inline Assets/MirageXR/Tests/NewUI/RootView_v2.cs]: `_button` -> `_btn`

**Review by cawke — CHANGES_REQUESTED** [review]

File: Assets/MirageXR/Tests/NewUI/RootView_v2.cs (hunk 4 above)
Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/RootView_v2.cs]: done

File: Assets/MirageXR/Tests/NewUI/ProfileView.cs (hunk 3 above)
Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/ProfileView.cs]: done

File: Assets/MirageXR/Tests/NewUI/Prefabs/Profile.prefab (hunk 2 above)
Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/Prefabs/Profile.prefab:523]: done

File: Assets/MirageXR/Tests/NewUI/RootView_v2.cs (hunk 1 above)
Comment (Alvareon) [reply Assets/MirageXR/Tests/NewUI/RootView_v2.cs]: done

**Review by AlexanderAShubin — APPROVED** [review]
Thanks!

**Review by cawke — APPROVED** [review]
Thanks!

**Pull request merged**